rotkehlchen/exchanges/bittrex.py

```diff
@@ -274,6 +276,16 @@ def _single_api_query(
         except requests.exceptions.ConnectionError as e:
             raise RemoteError(f'Bittrex API request failed due to {str(e)}')
 
+        # Error due to unsync system clock (only for non-public endpoints)
+        if not public_endpoint and response.status_code == HTTPStatus.UNAUTHORIZED:
+            json_ret = rlk_jsonloads_dict(response.text)
+            if json_ret.get('code') == 'INVALID_TIMESTAMP':
```

Comment:
Here you should add `get('code', None)`. Your current version will raise a `KeyError` if for some reason bittrex returns a json object without the `'code'` entry

Reply:
dict `.get()` defaults to `None` if the key is not found.

Reply:
ah yes indeed. Sorry disregard.

Reply:
Even though I like explicitness, but here as you said it's not needed.

Reply:
If you want it explicit I don't mind, really. I will add it. I've done it both ways before :)